extension-code/Collision-Tags.js

```diff
@@ -0,0 +1,139 @@
+// Name: Collision Tags
+// ID: lbgcollisiontag
+// Description: Basically just a touching with collision tag block, inspired by one of lily's ideas
+// Inspired by: An idea Lily had but didnt make yet
+// By: LittleBlueGamer
+
+// Version V.1.0.0
+
+(function(Scratch) {
+  "use strict";
+
+  const COLLISION_TAG = Symbol("collision.tag");
+
+  const vm = Scratch.vm;
+
+  const implementTag = (target, originalTarget) => {

```

Comment:
I dont suggest adding this to every target on project load/target created.

I'd rather you just check if a `lbgCOLLISION_TAG` key exists in the target json, then do whatever with it. It would also be easier to implement a remove/reset collision tag block

Reply:
oh shit

Reply:
(also its intended to have a default tag)

Reply:
I know, but it doesn't need all this stuffs, if it's undefined it can default to the default text